R/seq_linter.R

```diff
@@ -11,15 +15,25 @@
 seq_linter <- function() {
   bad_funcs <- c("length", "n", "nrow", "ncol", "NROW", "NCOL", "dim")
 
+  # Exact `xpath` depends on whether bad function was used in conjunction with `seq()`
+  bad_func_xpath_with_seq <- glue::glue(
+    "expr[1][SYMBOL_FUNCTION_CALL[text() = 'seq']]/following::expr[1]/expr[SYMBOL_FUNCTION_CALL[ {xp_text_in_table(bad_funcs)} ]]"
```

Comment:
@MichaelChirico I am open to suggestion on how this xpath can be changed to avoid the `line_length_linter` lint.